tests/test_ramp_fitting_cases.py

```diff
@@ -19,9 +20,17 @@
 #  their SCI values (these are mostly for my reference).
 #
 
+
+################## DEBUG ################## 
+#                  HELP!!
+import sys
+sys.path.insert(1, "/Users/kmacdonald/code/common")
+from general_funcs import *
```

Comment:
what is this for? what "functions" do you use from `general_funcs`?

Reply:
This should not be there.  Those are a set of functions I use for debugging.  I removed this import.